src/logtab.cpp

```diff
@@ -912,10 +897,9 @@ void LogTab::UpdateStatusBar()
             }
             status += highlightOpt.m_value;
         }
-        status += "}  |  ";
+        status += "}";
     }
 
-    status += QString("events: %1  ").arg(m_treeModel->rowCount());
```

Comment:
Synced up offline - we'll remove it. There's Summary dialog now you could use for this.